repl/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "hyperon_repl"
+version = "0.1.0"
```

Comment:
On crates.io, they can have independent versions - and we might want to publish updates separately to the core and the repl.

But from the user's perspective I see that we probably want to be clear when they run `metta --version` what version of the core they have as well as the repl.  I added the following todo to https://github.com/trueagi-io/hyperon-experimental/issues/398

`metta --version invocation should return both the core metta version as well as the repl version`

Reply:
Ok, publishing Rust is less complex than publishing Python I hope :-) Thus we can publish repl separately. But yes, we should print the version of the core library in `metta --version` otherwise it will be difficult to debug it.

Reply:
Publishing Rust is very very easy.  I have todo-items in the tasks issue to tackle the version issues, (both reporting the versions, as well as validating the hyperonpy version)

I am not worried about hyperon-core <-> repl version mismatches because both are managed by Cargo.  I am terrified of hyperonpy <-> core / repl version mismatches.